instrumentation/opentelemetry-api-1.0/javaagent/src/test/groovy/TracerTest.groovy

```diff
@@ -34,7 +34,7 @@ class TracerTest extends AgentInstrumentationSpecification {
           name "test"
           kind PRODUCER
           hasNoParent()
-          status ERROR
+          errored true
```

Comment:
Are you sure that we don't want to make a reverse change? Spec does not define `errored`, it defines `status` :)

Reply:
ya, that makes sense to me, will update the PR tomorrow

Reply:
@trask Do you plan to do this?

Reply:
this is done now, cleared existing reviews